content/cpp/concepts/loops/loops.md

```diff
@@ -42,4 +42,11 @@ This example prints 0 to 9 on the screen:
 for (int i = 0; i < 10; i++) {
   std::cout << i << "\n";
 }
+
+// This would also run the same as above.
+int i = 0;
+for( ; i < 10; ) {
+  std::cout << i << "\n";
+  i++;
```

Comment:
@bordenmike518 I would also like to see something about how we can add more than one condition and action.

`i++;` can go here or in the parenthesis.